components/Wallet/PlaceOrder/Form.jsx

```diff
@@ -143,7 +143,7 @@ export function PlaceOrderForm({ showTitle = true, asset, onSubmit, components:
     const amount = getAdjOrderAmount(currentState)
 
     // Amount should be based on asset decimals
-    // currentState.amount = formatFloat(amount, asset.decimals) || ''
+    currentState.amount = formatFloat(amount, asset.decimals) || ''
```

Comment:
I don't think this is a good solution - can you find a different one? We shouldn't be changing the amount field as you type it as it causes other weird behavior.